### ISSUE TITLE
`generate-zip-file` build task failing

### ISSUE THREAD

**Issue by anderseknert** [StyraInc/opa-aws-cloudformation-hook#49]
I think this is due to a more recent version of Python now coming with the runners, but I'm not sure. We should look into it.

Example: https://github.com/StyraInc/opa-aws-cloudformation-hook/actions/runs/8170632332

**Comment by PatMyron** [comment]
https://github.com/StyraInc/opa-aws-cloudformation-hook/actions/workflows/check-resource-types.yaml failing too

**Comment by anderseknert** [comment]
Yeah. Any clues as to what might have caused it? I’ve not had the time to look into this yet.
